tests/components/whirlpool/snapshots/test_sensor.ambr

```diff
@@ -153,6 +153,474 @@
     'state': 'running_maincycle',
   })
 # ---
+# name: test_all_entities[sensor.oven_lower_oven_cook_mode-entry]
+  EntityRegistryEntrySnapshot({
+    'aliases': set({
+    }),
+    'area_id': None,
+    'capabilities': dict({
+      'options': list([
+        'standby',
+        'bake',
+        'convection_bake',
+        'broil',
+        'convection_broil',
+        'convection_roast',
+        'keep_warm',
+        'air_fry',
+      ]),
+    }),
+    'config_entry_id': <ANY>,
+    'config_subentry_id': <ANY>,
+    'device_class': None,
+    'device_id': <ANY>,
+    'disabled_by': None,
+    'domain': 'sensor',
+    'entity_category': None,
+    'entity_id': 'sensor.oven_lower_oven_cook_mode',
+    'has_entity_name': True,
+    'hidden_by': None,
+    'icon': None,
+    'id': <ANY>,
+    'labels': set({
+    }),
+    'name': None,
+    'options': dict({
+    }),
+    'original_device_class': <SensorDeviceClass.ENUM: 'enum'>,
+    'original_icon': None,
+    'original_name': 'Lower oven cook mode',
+    'platform': 'whirlpool',
+    'previous_unique_id': None,
+    'supported_features': 0,
+    'translation_key': 'oven_cook_mode_lower',
+    'unique_id': 'said_oven-oven_cook_mode_lower',
+    'unit_of_measurement': None,
+  })
+# ---
+# name: test_all_entities[sensor.oven_lower_oven_cook_mode-state]
+  StateSnapshot({
+    'attributes': ReadOnlyDict({
+      'device_class': 'enum',
+      'friendly_name': 'Oven Lower oven cook mode',
+      'options': list([
+        'standby',
+        'bake',
+        'convection_bake',
+        'broil',
+        'convection_broil',
+        'convection_roast',
+        'keep_warm',
+        'air_fry',
+      ]),
+    }),
+    'context': <ANY>,
+    'entity_id': 'sensor.oven_lower_oven_cook_mode',
+    'last_changed': <ANY>,
+    'last_reported': <ANY>,
+    'last_updated': <ANY>,
+    'state': 'unknown',
```

Comment:
something is not working since this should have a value

Reply:
Fixed